src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -98,7 +97,12 @@ def on_time_step(self, event: Event) -> None:
             p=move_type_probabilities.values,
         )
 
-        domestic_movers_idx = pop.index[move_types_chosen != "no_move"]
+        pop = self._perform_new_household_moves(
```

Comment:
Conventionally `pop` means the "whole" pop table. I'd suggest storing the result of this call in a new variable.for clarity.

Reply:
This is the whole pop table, unless I am misunderstanding what you mean by "whole". Would you prefer that this method returned only the movers?

Reply:
Sorry, disregard my comment. I must have misread what is happening in the `perform_hew_household_Moves` method.